src/herder/TxSetFrame.h

```diff
@@ -47,14 +47,14 @@ class TxSetFrame : public NonMovableOrCopyable
     // **Note**: the output `TxSetFrame` will *not* contain the input
     // transaction pointers.
     static TxSetFrameConstPtr
-    makeFromTransactions(TxPhases const& txPhases, Application& app,
-                         uint64_t lowerBoundCloseTimeOffset,
-                         uint64_t upperBoundCloseTimeOffset);
-    static TxSetFrameConstPtr
     makeFromTransactions(TxPhases const& txPhases, Application& app,
                          uint64_t lowerBoundCloseTimeOffset,
                          uint64_t upperBoundCloseTimeOffset,
-                         TxPhases& invalidTxsPerPhase);
+                         bool forceIsNotGeneralized = false);
+    static TxSetFrameConstPtr makeFromTransactions(
```

Comment:
I think the issue is in the test, which is creating tx sets rather than generalized tx sets, so feel free to revert this change.

Reply:
Sorry I thought we're meant to create tx sets rather than generalized tx sets, as the node is meant to receive the tx set? My recollection is that generalized tx set basically just contains the hash?

Reply:
generalized transaction sets are also transactions sets, but with a slightly different structure to accommodate protocol 20. See definition here: https://github.com/stellar/stellar-xdr/blob/curr/Stellar-ledger.x#L234.